Tests/NetworkServiceTests/NetworkServiceTests.swift

```diff
@@ -1,8 +1,114 @@
-import NetworkService
+@testable import NetworkService
```

Comment:
why do we need testable here?

Reply:
To gain access to NetworkService in `func setUpWithError()`

Reply:
I think we have to add public `init` to be able to create an instance of `NetworkService` when the package is added to a project.

Reply:
@kirpichonok thanks for comment!
Fixed with c152386